packages/frontend/src/pages/MobileHome.tsx

```diff
@@ -0,0 +1,133 @@
+
+    const {
+        data: mostPopularAndRecentlyUpdated,
+        isLoading: isMostPopularAndRecentlyUpdatedLoading,
+    } = useMostPopularAndRecentlyUpdated(selectedProjectUuid);
+
+    const { user } = useApp();
+    const items = useMemo(() => {
+        const mostPopularItems =
+            mostPopularAndRecentlyUpdated?.mostPopular.map((item) => ({
+                ...wrapResource(
+                    item,
+                    'chartType' in item
+                        ? ResourceViewItemType.CHART
+                        : ResourceViewItemType.DASHBOARD,
+                ),
+                category: ResourceItemCategory.MOST_POPULAR,
+            })) ?? [];
+        const pinnedItemsWithCategory = pinnedItems.map((item) => ({
+            ...item,
+            category: ResourceItemCategory.PINNED,
+        }));
+        return [...pinnedItemsWithCategory, ...mostPopularItems];
+    }, [mostPopularAndRecentlyUpdated, pinnedItems]);
+
+    const isLoading =
+        project.isLoading ||
+        savedChartStatus.isLoading ||
+        isMostPopularAndRecentlyUpdatedLoading ||
+        pinnedItemsLoading;
+    const error = project.error || savedChartStatus.error;
+
+    if (user.data?.ability?.cannot('view', 'SavedChart')) {
+        return <ForbiddenPanel />;
+    }
+
+    if (isLoading) {
+        return <PageSpinner />;
+    }
+
+    if (error) {
+        return <ErrorState error={error.error} />;
+    }
+
+    if (!project.data) {
+        return <ErrorState />;
+    }
+
+    return (
+        <Stack spacing="md" m="lg">
+            <Stack justify="flex-start" spacing="xs">
+                <Title order={3}>
+                    {`Welcome${
+                        user.data?.firstName
+                            ? ', ' + user.data?.firstName
+                            : ' to Lightdash'
+                    }!`}{' '}
+                    ⚡️
+                </Title>
+            </Stack>
+            <ResourceView
+                items={items}
+                tabs={
+                    pinnedItems.length > 0
+                        ? [
+                              {
+                                  id: 'pinned',
+                                  name: 'Pinned',
+                                  filter: (item) =>
+                                      'category' in item &&
+                                      item.category ===
+                                          ResourceItemCategory.PINNED,
+                              },
+                              {
+                                  id: 'most-popular',
+                                  name: 'Most popular',
+                                  filter: (item) =>
+                                      'category' in item &&
+                                      item.category ===
+                                          ResourceItemCategory.MOST_POPULAR,
+                              },
```

Comment:
do we also want `recently updated` here?

Reply:
I decided to leave it out on purpose. So it's less crowded and because mobile users will most likely just want to access important charts rather than explore data.

Reply:
Makes sense!